### ISSUE TITLE
Change default mode that gets turned on

### ISSUE THREAD

**Issue by pbassham** [pdlove/homebridge-sensibo#10]
Is there a way to change the default mode that turns on when I turn on when I say "turn on ACCESSORY_NAME"?

Right now it doesn't really turn on my ac's, it just turns on fans, which don't really cool down rooms.

**Comment by pdlove** [comment]
Not really. You should be able to say "Set ACCESSORY_NAME to cool." Or something similar.

**Comment by pbassham** [comment]
"Turn Living Room on Cool" seems to work. 

"Set..." doesn't seem to though. Thanks.
